scripts/InitialCompute.py

```diff
@@ -0,0 +1,51 @@
+from NBA import getGames
+from NBA.models import Game,Historical_team_elo,Current_team_elo
+import os
+
+def write_games():
+    date_range = ["2000-00-00","9999-99-99"]
+    teams_df = getGames.get_team_games(date_range)
+    print(teams_df)
+    for indx,row in teams_df.iterrows():
+        object = Game(*(row.tolist()))
+        object.save()
+
+def elo_diff(elo1,elo2,wl):
+    p1_win = 1/(1+10**((elo2-elo1)/400))
+    if wl == "W":
```

Comment:
If you have your own Game object, this function should take in 2 Teams (which includes their elo) and a Game then update their Elo and the Players Elo accordingly after.